node/configs/src/client.rs

```diff
@@ -9,6 +9,7 @@ use primitives::types::AccountId;
 const DEFAULT_BASE_PATH: &str = ".";
 const DEFAULT_LOG_LEVEL: &str = "Info";
 const MAX_NUM_BLOCKS_REQUEST: u64 = 100;
+const MAX_BLOCK_SIZE: u32 = 1000;
```

Comment:
After load testing our network, 10k looks more reasonable.
```suggestion
const MAX_BLOCK_SIZE: u32 = 10000;
```